demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -480,18 +480,34 @@ svg {
   }
 
   .video-preview {
```

Comment:
Were you able to test these on multiple devices? I'm seeing px being used which makes me think this wont be responsive. Have you tried this on small mobile devices and also widescreen monitors?

Reply:
Yes I tested my changes on multiple screen sizes.